database.cc

```diff
@@ -1024,34 +1024,26 @@ std::vector<lw_shared_ptr<column_family>> database::get_non_system_column_famili
 }
 
 column_family& database::find_column_family(std::string_view ks_name, std::string_view cf_name) {
-    try {
-        return find_column_family(find_uuid(ks_name, cf_name));
-    } catch (...) {
-        std::throw_with_nested(no_such_column_family(ks_name, cf_name));
-    }
+    return find_column_family(find_uuid(ks_name, cf_name));
```

Comment:
As I explained in https://github.com/scylladb/scylla/issues/9754#issue-1073511639
I think it's fine to return the bad param error initially when validating the request parameters and then, if the table goes away, or for any other reason we fail to locate it - it's indeed an internal server error (that doesn't care to lock anything while performing the api call)

Reply:
since these calls are synchronous, it should be an internal error to not find the uuid returned by `find_uuid(ks_name, cf_name)`.  However, currently find_column_family(uuid) throws no_such_column_family(uuid) which is not the best error since it hides the original ks_name, cf_name; but I think we can live with it.

Reply:
Done in #9755